tutormfe/templates/mfe/tasks/lms/init

```diff
@@ -100,3 +100,9 @@ grep openresponseassessment.enhanced_staff_grader /tmp/lms_waffle_flags.txt || .
 {% else %}
 ./manage.py lms waffle_delete --flags openresponseassessment.enhanced_staff_grader
 {% endif %}
+
+{% if is_mfe_enabled("catalog") %}
```

Comment:
TODO/info: we plan to add several waffle flags (one for each page) after the BE part is developed and the DEPR process is complete, the changes will be added to the PR.